client/app/components/EditParameterSettingsDialog.jsx

```diff
@@ -143,7 +172,7 @@ function EditParameterSettingsDialog(props) {
             <Option value="datetime-range-with-seconds">Date and Time Range (with seconds)</Option>
           </Select>
         </Form.Item>
-        {includes(['date', 'datetime-local', 'datetime-with-seconds'], param.type) && (
+        {isTypeDate(param.type) && (
```

Comment:
I don't know why, but it works only for `date` type:

![image](https://user-images.githubusercontent.com/71468/53861911-42621000-3fee-11e9-8880-179b06ba15e7.png)

(notice that there is no checkbox)

Reply:
😅 Mercy fire me plz.

Fixed https://github.com/getredash/redash/pull/3528/commits/7d6e0d8f8c86265aa9e06c481a3be4cbf5cd6f04